test/functional/cmdLineTests/valuetypeddrtests/flattened32bitRefBackfillTest.xml

```diff
@@ -118,7 +118,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<input>!fj9object $flatSingleBackfillInstanceO$</input>
 			<input>quit</input>
 		</command>
-		<output regex="no" type="success" showMatch="yes">I value = 0xEEFFEEFF (offset = 4) (java/lang/Integer)</output>
+		<output regex="no" type="success" showMatch="yes">I value = 0xEEFFEEFF (offset = 0) (java/lang/Integer)</output>
```

Comment:
The commit message is "Update flattened primitive layouts for valuetypeddrtests". Please add some detail on why the offset is changed from 4 to 0.  (Is that because the lockword is gone after Integer becomes VT ? )

Reply:
> Is that because the lockword is gone after Integer becomes VT ?

Yes. These tests are accessing a field of type `java.lang.Object` that is initialized with the value ValueTypeTests.defaultObject which is an Integer.
```
static Object defaultObject = (Object)0xEEFFEEFF;
```
I will update the commit message to add more details